west.yml

```diff
@@ -101,7 +101,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: de87b6a5ee7a80cd1b6e0a549f896ba7f921bb59
+      revision: pull/475/head
```

Comment:
We should probably not use this revision, but wait for the revision to comming from nrfxlib.

Reply:
Correct, this is temporary to run CI before nrfxlib is merged. And then this can be updated.